msg.proto

```diff
@@ -99,4 +107,22 @@ message Msg {
   bytes write_file_sync_data = 131;
   uint32 write_file_sync_perm = 132;
   // write_file_sync_perm specified by https://godoc.org/os#FileMode
+
+  // HTTP_CREATE
+  int32 http_server_id = 133;
+
+  // HTTP_LISTEN
+  int32 http_listen_port = 134;
+
+  // HTTP_REQ
+  int32 http_req_id = 135;
+  string http_req_method = 136;
+  string http_req_path = 137;
+  bytes http_req_body = 138;
+
+  // HTTP_RES
+  int32 http_res_id = 139;
+  bytes http_res_body = 140;
+  int32 http_res_code = 141;
+  repeated string http_res_headers = 142;
```

Comment:
In the future, I'm hoping to reuse the structure for FETCH_RES for server messages (by adding a new message type) . But no changes requested now.

Reply:
Sounds fine